plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakePageSourceProvider.java

```diff
@@ -115,7 +115,6 @@ public class DeltaLakePageSourceProvider
     private final FileFormatDataSourceStats fileFormatDataSourceStats;
     private final ParquetReaderOptions parquetReaderOptions;
     private final int domainCompactionThreshold;
-    private final DateTimeZone parquetDateTimeZone;
```

Comment:
Let's leave the removal of `delta.parquet.time-zone` for a follow-up PR.
Pls drop the current commit.